zuul.d/content_provider.yaml

```diff
@@ -15,19 +15,38 @@
       - ^tests/.*$
       - ^kuttl-test.yaml$
     required-projects:
-      - opendev.org/zuul/zuul-jobs
-      - openstack-k8s-operators/install_yamls
-      - openstack-k8s-operators/openstack-operator
       - github.com/openstack-k8s-operators/ci-framework
```

Comment:
I was about to say to remove "github.com" from this one e repo-setup, but you already noticed that those 2 are duplicated here, so we can remove them.

Reply:
Done, thx.